chapter2/tests/t19/terraform/base/infra.auto.tfvars

```diff
@@ -0,0 +1,18 @@
+client    = "maxim-omelchenko"
```

Comment:
add doc blocks the same you did in dev.tfvars to separate the var groups

Reply:
> add doc blocks the same you did in dev.tfvars to separate the var groups

@dmitry-mightydevops
Done